<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>701</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="mainWindowLayout" rowstretch="0,0,0,0" columnstretch="0,0,0">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item row="1" column="2">
      <widget class="QPushButton" name="fillButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../data/data.qrc">
         <normaloff>:/pics/Paintcan.png</normaloff>:/pics/Paintcan.png</iconset>
       </property>
       <property name="shortcut">
        <string>Ctrl+3</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="circleButton">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../data/data.qrc">
         <normaloff>:/pics/Select-Ellipse.png</normaloff>:/pics/Select-Ellipse.png</iconset>
       </property>
       <property name="shortcut">
        <string>Ctrl+6</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="brushButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../data/data.qrc">
         <normaloff>:/pics/Painbrush.png</normaloff>:/pics/Painbrush.png</iconset>
       </property>
       <property name="shortcut">
        <string>Ctrl+1</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="eraserButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../data/data.qrc">
         <normaloff>:/pics/Draw-Eraser.png</normaloff>:/pics/Draw-Eraser.png</iconset>
       </property>
       <property name="shortcut">
        <string>Ctrl+2</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="rectButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../data/data.qrc">
         <normaloff>:/pics/Select-Restangular.png</normaloff>:/pics/Select-Restangular.png</iconset>
       </property>
       <property name="shortcut">
        <string>Ctrl+5</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="lineButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../data/data.qrc">
         <normaloff>:/pics/Draw-Line.png</normaloff>:/pics/Draw-Line.png</iconset>
       </property>
       <property name="shortcut">
        <string>Ctrl+4</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QWidget" name="widget" native="true"/>
     </item>
     <item row="0" column="0" rowspan="4">
      <widget class="QTabWidget" name="tabWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>279</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="tabsClosable">
        <bool>true</bool>
       </property>
       <property name="movable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="tab1">
        <attribute name="title">
         <string>tab1</string>
        </attribute>
        <widget class="DrawArea" name="widget_2" native="true">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>601</width>
           <height>231</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>702</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_File"/>
    <addaction name="actionOpen_File"/>
    <addaction name="separator"/>
    <addaction name="actionSave_File"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="actionOptions"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCopy"/>
    <addaction name="actionCut"/>
    <addaction name="actionPaste"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew_File">
   <property name="icon">
    <iconset resource="../data/data.qrc">
     <normaloff>:/pics/Document-Empty.png</normaloff>:/pics/Document-Empty.png</iconset>
   </property>
   <property name="text">
    <string>New File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen_File">
   <property name="icon">
    <iconset resource="../data/data.qrc">
     <normaloff>:/pics/Folder.png</normaloff>:/pics/Folder.png</iconset>
   </property>
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
  <action name="actionSave_File">
   <property name="icon">
    <iconset resource="../data/data.qrc">
     <normaloff>:/pics/Disk.png</normaloff>:/pics/Disk.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset resource="../data/data.qrc">
     <normaloff>:/pics/Save-As.png</normaloff>:/pics/Save-As.png</iconset>
   </property>
   <property name="text">
    <string>Save As ...</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="icon">
    <iconset resource="../data/data.qrc">
     <normaloff>:/pics/Document-Export.png</normaloff>:/pics/Document-Export.png</iconset>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="icon">
    <iconset resource="../data/data.qrc">
     <normaloff>:/pics/Setting-Tools.png</normaloff>:/pics/Setting-Tools.png</iconset>
   </property>
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../data/data.qrc">
     <normaloff>:/pics/Traffic-Lights-Red.png</normaloff>:/pics/Traffic-Lights-Red.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="../data/data.qrc">
     <normaloff>:/pics/Arrow-Undo.png</normaloff>:/pics/Arrow-Undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="../data/data.qrc">
     <normaloff>:/pics/Arrow-Redo.png</normaloff>:/pics/Arrow-Redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="../data/data.qrc">
     <normaloff>:/pics/Page-White-Copy.png</normaloff>:/pics/Page-White-Copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="../data/data.qrc">
     <normaloff>:/pics/Cut.png</normaloff>:/pics/Cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="../data/data.qrc">
     <normaloff>:/pics/Page-White-Paste.png</normaloff>:/pics/Page-White-Paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="../data/data.qrc">
     <normaloff>:/pics/Help.png</normaloff>:/pics/Help.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DrawArea</class>
   <extends>QWidget</extends>
   <header>drawarea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../data/data.qrc"/>
 </resources>
 <connections/>
</ui>
